tests/unit/core/test_docs_util.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding:utf-8 -*-
```

Comment:
Declaring UTF-8 isn't needed unless there are non-ascii chars in this file, which there are not.